evap/staff/importers/person.py

```diff
@@ -0,0 +1,100 @@
+from typing import Iterable
+
+from django.utils.html import format_html
+from django.utils.translation import gettext as _
+
+from evap.evaluation.models import Contribution, Evaluation, UserProfile
+from evap.staff.tools import ImportType, create_user_list_html_string_for_message
+
+from .base import ImporterLog
+from .user import import_users
+
+
+def add_participants_to(
+    evaluation: Evaluation, users: Iterable[UserProfile], test_run: bool, importer_log: ImporterLog
+):
+    evaluation_participants = evaluation.participants.all()
+    already_related = [user for user in users if user in evaluation_participants]
+    users_to_add = [user for user in users if user not in evaluation_participants]
```

Comment:
Searching for other places where we could use that method using grep with
```regexp
for.*in.*if.*not.*in
```
gives me no other results. Since we're talking about two lines, I'd argue we should leave it inlined until we'd have at least one more usage.

Reply:
Sure